src/kibana/components/doc_table/doc_table.less

```diff
@@ -7,6 +7,12 @@ doc-table {
   margin: 5px;
   .flex(1, 1, 100%);
 
+  tr.discover-table-row  {
+    td:not(.discover-table-timefield):not(.ng-scope) {
```

Comment:
Assuming those templates are run through the no white space utility it shouldn't be a problem.